app/controllers/SummaryController.scala

```diff
@@ -45,26 +44,23 @@ class SummaryController @Inject() (
   def show(taxYear: Int): Action[AnyContent] = (identify andThen getData).async { implicit request =>
     withUpdatedData(taxYear) {
       val hc = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
-      businessService.getBusinessDetails(request.user)(hc).map {
-        case Right(businessDetails) if businessDetails.propertyData.exists(existsUkProperty) =>
-          val propertyData = businessDetails.propertyData.find(existsUkProperty).get
+      businessService.getUkPropertyDetails(request.user.nino, request.user.mtditid)(hc).flatMap {
+        case Right(Some(propertyData)) =>
           val propertyRentalsRows =
             SummaryPage.createUkPropertyRows(request.userAnswers, taxYear, propertyData.cashOrAccruals.get)
```

Comment:
There are some .gets, could you fix that too?

Reply:
theres no .get

Reply:
Nope will not fix that, thats extra for this PR